pl_bolts/datamodules/cifar10_datamodule.py

```diff
@@ -58,136 +54,75 @@ class CIFAR10DataModule(LightningDataModule):
         dm.val_transforms  = ...
     """
 
-    name = 'cifar10'
-    extra_args = {}
+    if not _TORCHVISION_AVAILABLE:
+        raise ModuleNotFoundError(  # pragma: no-cover
+            "You want to use CIFAR10 dataset loaded from `torchvision` which is not installed yet."
+        )
```

Comment:
@Borda, @akihironitta, how can I fix the CI base testing tests without torchvision? I thought moving up the error raising would help but it doesn't.

Reply:
you need to 
```
if _TORCHVISION_AVAILABLE:
    from torchvision... import MNIST
else:
    MNIST = None
```

Reply:
Thanks!